### PR TITLE
remove GS Collections

### PR DESCRIPTION
**Pull request by vborisoff**

### CODE REVIEW

File: reladomo/src/main/java/com/gs/fw/finder/attribute/TimestampAttribute.java [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
+
+import com.gs.fw.finder.Attribute;
+import com.gs.fw.finder.Operation;
+
+import java.sql.Timestamp;
+import java.util.Set;
+
+public interface TimestampAttribute<Owner> extends Attribute<Owner>
+{
+    Operation<Owner> eq(Timestamp value);
```
Comment (mohrezaei) [inline reladomo/src/main/java/com/gs/fw/finder/attribute/TimestampAttribute.java]: let's not mix this with more edits (it's big enough as it is)

**Comment by mohrezaei** [comment]
The PR is mostly good, but it might be better to start with the JDK 8 stuff so that we can the continuous build to work again first.

**Comment by motlin** [comment]
It looks like every line is changed. Did you change the newlines?

FWIW, when I work on Reladomo, I run `git config core.autocrlf false` first.

There's plenty of debate about the right autocrlf setting [on StackOverflow](https://stackoverflow.com/questions/2825428/why-should-i-use-core-autocrlf-true-in-git).

The documentation at git-scm and [github](https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings) recommend `true` on windows and `input` elsewhere. Probably because of these docs, I find that `false` is a pretty rare choice and Reladomo is the only project I work on where I have to remember to configure autocrlf differently than my `~/.gitconfig`

Once the newlines are sorted, I'm eager to see the actual change.

**Comment by mohrezaei** [comment]
@motlin You can press the settings button in the review tab and ignore whitespace changes. I'm using an `.editorconfig` file in another project and it helps. I think these settings would work for Reladomo:
```
root = true

[*]
charset = utf-8
end_of_line = crlf
indent_size = 4
continuation_indent_size = 8
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true
```

**Comment by mohrezaei** [comment]
settings

**Comment by motlin** [comment]
Thanks! I managed to view the diffs locally. First I added pr refs to those that I fetch.

```ini
[remote "upstream"]
    url = https://github.com/goldmansachs/reladomo.git
    fetch = +refs/heads/*:refs/remotes/upstream/*
    fetch = +refs/pull/*:refs/remotes/upstream/pr/*
```

Then I fetched everything.
```bash
git fetch --all
```

Then I "showed" the pr ignoring whitespace.

```bash
git show upstream/pr/208/head -w
```

This should still be fixed before merging the changes, or all these line endings will change from CRLF to LF, and it will mess with vcs history.

File: reladomo/src/main/java/com/gs/fw/finder/attribute/TimestampAttribute.java (hunk 1 above)
Comment (vborisoff) [reply reladomo/src/main/java/com/gs/fw/finder/attribute/TimestampAttribute.java]: will do.
 BTW, you mentioned that you have a draft of the JDK8 migration. Please share the branch.

**Comment by mohrezaei** [comment]
@vborisoff https://github.com/gs-rezaem/reladomo/tree/jdk18

**Comment by mohrezaei** [comment]
Two things for this commit:
1) Fix your git settings so that build.xml is not re-written.
2) Fix the travis config so that the build runs.

**Pull request closed without merging**